client/js/bar.js

```diff
@@ -0,0 +1,35 @@
+/* constructs bar graph from scratch */
+
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+svg.setAttribute('viewBox', '0 0 100 100'); // change height in css
+document.getElementById('graph').appendChild(svg);
+
+// hard coded data for bar chart for now
+const counts = [1, 3, 1, 2];
+const maxCount = 3;
+
+const GRAPH_HEIGHT = 100;
+const BAR_CONTAINER_HEIGHT = GRAPH_HEIGHT / counts.length;
+
+// bar container contains a bar. It is composed of bar padding and fill
+const BAR_PERCENT_FILL = 0.8;
+const BAR_PERCENT_PADDING = 1 - BAR_PERCENT_FILL;
+const BAR_THICKNESS = BAR_PERCENT_FILL * BAR_CONTAINER_HEIGHT;
+const BAR_UNIT_LENGTH = 100 / maxCount;
```

Comment:
Know we talked about this already, but this actually looks very clean and is a good way to set these values.